src/clojure/nrepl/middleware/pr_values.clj

```diff
@@ -27,6 +27,13 @@
                                       (dissoc resp :printed-value)
                                       (if-let [[_ v] (find resp :value)]
                                         (assoc resp
+                                               :value2 (let [repr (java.io.StringWriter.)]
+                                                         (apply
+                                                          (if renderf
+                                                            (eval (read-string renderf))
```

Comment:
Seems to me that `renderf` (or whatever we end up calling it) should be a symbol that we'd resolve and apply. Passing a lambda here doesn't seem very wise to me.